packages/design/src/icons/Colors.css

```diff
@@ -41,3 +41,11 @@
 .orange {
   fill: var(--color-orange);
 }
+
```

Comment:
@darryltec while we're in here, can we add support for _all_ of the semantic colors here?

Reply:
for you @chris-at-jobber ? Of course!

Reply:
@chris-at-jobber Tis' has been added